### PR TITLE
Whitelist CS withdrawal

### PR DESCRIPTION
**Pull request by aubergemediale**

### CODE REVIEW

File: src/Obsidian.Networks.ObsidianX/Rules/ObsidianXRequireNativeSegWitRule.cs [hunk 1]
```diff
@@ -17,29 +17,24 @@ public override Task RunAsync(RuleContext context)
 
             foreach (var tx in block.Transactions)
             {
-                if (tx.IsCoinStake) // Coinstake tx are custom to ObsidianX, skip
+                if (tx.IsCoinStake || tx.IsCoinBase)
```
Comment (dangershony) [inline src/Obsidian.Networks.ObsidianX/Rules/ObsidianXRequireNativeSegWitRule.cs]: Just use .IsProtocolTransaction

Comment (aubergemediale) [reply src/Obsidian.Networks.ObsidianX/Rules/ObsidianXRequireNativeSegWitRule.cs]: done!

**Comment by dangershony** [comment]
For the sake of testing on current testnet we can add a block height to the rule after with its activated.

However once I am done with the time field PR we just reset the network (or move to the new network)

**Comment by aubergemediale** [comment]
Closing this because I merged it in the other PR #40

**Pull request closed without merging**